services/cognito/config/routes.rb

```diff
@@ -5,4 +5,6 @@
   jsonapi_resources :users
   jsonapi_resources :pools
   jsonapi_resources :endpoints
+
+  resources :token, only: :show
```

Comment:
lets make this clearer. Since we have this in cognito token is not conveying the right message. I suggest we rename it to something like `metabase_token`

Reply:
ok